### ISSUE TITLE
Config not merging correctly when using full-config by tailwind

### ISSUE THREAD

**Issue by aislanmaia** [studiometa/tailwind-config#34]
I'm testing this with a project started out from vue-cli. 
I have added the tailwind config with ` npx tailwindcss init --full` command and build some screen to test out things like flexbox, grid and etc using tailwind classes. 

Now I've found out a plugin from this package to handle more easily grids, but I can't merge this config following the documentation.

I'm getting an error related to `.grid` class and @apply after put in the code to merge my configs with the config from this package. That's because my grid classes isn't found anymore.

This is what I'm trying to do in my `tailwind.config.js`:

```
// By merging the package's configuration with you own custom one
const config = require("@studiometa/tailwind-config");
const _ = require("lodash");

module.exports = _.merge(config, {
  future: {
    // removeDeprecatedGapUtilities: true,
    // purgeLayersByDefault: true,
  },
  purge: ["./public/**/*.html", "./src/**/*.html", "./src/**/*.vue"],
  target: "relaxed",
  prefix: "",
  important: false,
  separator: ":",
  presets: [],
  theme: {
    screens: {
      sm: "640px",
      md: "768px",
      lg: "1024px",
      xl: "1280px"
    },

    colors: {
      transparent: "transparent",
      current: "currentColor",    
/* ...etc */
```
But getting that error: 
![print-error](https://user-images.githubusercontent.com/2199137/98483511-b99a6f00-21e7-11eb-9b44-b6988b6655c7.jpg)

**Comment by perruche** [comment]
Hello,

The package is not using the `.grid` class from tailwind. It's a custom plugin that is use [see here](https://tailwind-config.meta.fr/plugins/grid.html#configuration) to configure it. 

If you want to be able to use `.grid`class with `@apply`, you will need to activate the experimental `applyComplexClasses` feature from tailwind available since [1.7.0](https://blog.tailwindcss.com/tailwindcss-1-7)

```js
module.exports = {
  experimental: {
    applyComplexClasses: true,
  },
};

**Comment by titouanmathis** [comment]
Hi @aislanmaia, 

To complete @perruche's answer, if you only want to use our grid plugin, you can directly import it in your configuration without using all of our specific config choices:

```js
const gridPlugin = require('@studiometa/tailwind-config/src/plugins/grid');

module.exports = {
  theme: {
    gridPlugin: {
      // Add a custom prefix to avoid conflict with Tailwind's own `.grid` class
      prefixValue: 'something',
    },
  },
  plugins: [
    gridPlugin(),
  ],
};
```

I think that with our 2 comments your issue should be resolved, I am closing this for now.
